.pre-commit-config.yaml

```diff
@@ -48,9 +48,9 @@ repos:
       log_file: '.artifacts/flake8.pycodestyle.log'
     - id: lint-requirements
       name: lint-requirements
-      entry: bin/lint-requirements
+      entry: python -m tools.lint_requirements sentry
```

Comment:
Mmm, yeah, I was thinking to just move this into CI rather than have it as a pc hook. I'll move the packaging dep to CI as well... or actually do you think it would be better to have a requirements file inside tools for now?

The freezer is stdlib only.

Reply:
yeah let's put this in CI instead 👍